core/migrations/0080_user_country_user_user_type.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2 on 2024-04-04 07:58
```

Comment:
Generally we should also create some sort of data migration for the existing users; but not sure if this is the case now.

@mereualexandra do we have any existing real staging/production users? Should we make them all "secretariat"?

Reply:
@raressiteavu @mereualexandra with Alexandra's and Anton's help, found there is a grand total of 3 users in production. It's ok to set their roles via admin. Will resolve this conversation. But generally speaking, for Rares, always think about that.